modules/calib3d/src/stereosgbm.cpp

```diff
@@ -2345,7 +2345,7 @@ void filterSpecklesImpl(cv::Mat& img, int newVal, int maxSpeckleSize, int maxDif
     using namespace cv;
 
     int width = img.cols, height = img.rows, npixels = width*height;
-    size_t bufSize = npixels*(int)(sizeof(Point2s) + sizeof(int) + sizeof(uchar));
+    size_t bufSize = npixels*(sizeof(Point2s) + sizeof(int)) + (npixels + 1)*sizeof(uchar);
```

Comment:
I propose to get rid of braces in the first case,  and add a comment to each sum part to mark corresponding buffer name.